src/runtime_src/core/edge/user/device_linux.cpp

```diff
@@ -42,6 +42,47 @@ get_edgedev(const xrt_core::device* device)
   return zynq_device::get_dev();
 }
 
+struct bdf 
+{
+  using result_type = query::pcie_bdf::result_type;
+
+  static result_type
+  get(const xrt_core::device* device, key_type)
+  {
+    return std::make_tuple(0,0,0);
```

Comment:
Perhaps
```suggestion
    return {0,0,0};
```
works?

Reply:
it works :)

Reply:
No, it doesn't work with older compilers, e.g. gcc version 5.4.0.  I am reverting.

Reply:
People who do not care about modern current software could use old versions of XRT and old FPGA anyway...
At the end there is a technical debt we all pay just because a lot of irresponsible ignorant managers all over the place... :-(